saleor/graphql/schema.graphql

```diff
@@ -5623,6 +5643,7 @@ type ProductVariant implements Node & ObjectWithMetadata {
   digitalContent: DigitalContent
   stocks(address: AddressInput, countryCode: CountryCode): [Stock]
   quantityAvailable(address: AddressInput, countryCode: CountryCode): Int!
+  preorder: PreorderData!
```

Comment:
@Kwaidan00 Frontend team asks if we cannot make this field nullable and when variant is not in preorder then we should return null here. I think it makes sense, because now they need to check `variant.preorder.isPreorder` and it should be enough just to check the presence of `variant.preorder`, what do you think?

Reply:
I have no strong feelings here, I can implement whatever is easier to use in API. I'll change it if they prefer it.